AuthenticationSystem/templates/start.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome</title>
+</head>
+<body>
```

Comment:
Remove this html page, Right now logout, page is not need, instead provide it as function to call it in profile page or settings page.